insightvm_connector.py

```diff
@@ -449,28 +433,37 @@ def main():
     if username is not None and password is None:
         # User specified a username but not a password, so ask
         import getpass
+
         password = getpass.getpass("Password: ")
 
     if username and password:
         try:
-            login_url = InsightVMConnector._get_phantom_base_url() + '/login'
+            login_url = InsightVMConnector._get_phantom_base_url() + "/login"
 
             print("Accessing the Login page")
-            r = requests.get(login_url, timeout=consts.INSIGHTVM_DEFAULT_TIMEOUT, verify=True)
-            csrftoken = r.cookies['csrftoken']
+            r = requests.get(
+                login_url, timeout=consts.INSIGHTVM_DEFAULT_TIMEOUT, verify=True
+            )
+            csrftoken = r.cookies["csrftoken"]
 
             data = dict()
-            data['username'] = username
-            data['password'] = password
-            data['csrfmiddlewaretoken'] = csrftoken
+            data["username"] = username
+            data["password"] = password
+            data["csrfmiddlewaretoken"] = csrftoken
 
             headers = dict()
-            headers['Cookie'] = 'csrftoken=' + csrftoken
-            headers['Referer'] = login_url
+            headers["Cookie"] = "csrftoken=" + csrftoken
+            headers["Referer"] = login_url
 
             print("Logging into Platform to get the session id")
-            r2 = requests.post(login_url, verify=True, data=data, headers=headers, timeout=consts.INSIGHTVM_DEFAULT_TIMEOUT)
-            session_id = r2.cookies['sessionid']
+            r2 = requests.post(
+                login_url,
+                verify=True,
+                data=data,
+                headers=headers,
+                timeout=consts.INSIGHTVM_DEFAULT_TIMEOUT,
+            )
+            session_id = r2.cookies["sessionid"]
         except Exception as e:
             print("Unable to get session id from the platform. Error: {}".format(str(e)))
             exit(1)
```

Comment:
Use `sys.exit()` instead of `exit()` whenever applicable.

Reply:
updated